### ISSUE TITLE
[FP] Suspicious argument with switch

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1226]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 2.0.13 |

Current behaviour
--

This case is a voluntary fall through with a switch. This triggers an error "Overrides value from a preceding case (perhaps a 'break' is missing there)." But it is wanted and it's not overrided because we use the first value to calculate the second one

```php
<?php
$bar = 'b';
switch ($foo) {
    case 1:
        $bar = 'a';
    case 2:
    default:
        $bar = trim($bar);
        break;
}
```

**Comment by kalessil** [comment]
Fixed!
